dev/deps/spark-deps-hadoop-2.2

```diff
@@ -115,7 +115,7 @@ libfb303-0.9.2.jar
 libthrift-0.9.2.jar
 log4j-1.2.17.jar
 lz4-1.3.0.jar
-mesos-0.21.1-shaded-protobuf.jar
+mesos-0.22.2-shaded-protobuf.jar
```

Comment:
These files are part of the test suite. The tests will fail if the dependencies in these files don't match the deps in the pull request. At least thats my interpretation, whats the usual way of suggestion dependency updated in a pull request?